Nanoleaf-Api/Endpoints/EffectsEndpoint.cs

```diff
@@ -38,8 +38,8 @@ public Task<string> GetSelectedEffectAsync()
 	    /// <inheritdoc />
 	    public string GetSelectedEffect()
 		{
-			return GetSelectedEffectAsync().ConfigureAwait(false).GetAwaiter().GetResult();
-		}
+            throw new NotImplementedException();
```

Comment:
Why remove the functionality instead of calling SendRequest?

Reply:
Because we do not use it yet plus it is not completely clear how to retrieve it